docs/model_library/strategic_water_management/index.rst

```diff
@@ -1049,30 +951,22 @@ The total reuse volume is the total volume of produced water reused, or the tota
 .. math::
 
     \textcolor{red}{F^{TotalCompletionsReused}}
-        = \sum_{t \in T}(\sum_{(n,p) \in NCA}\textcolor{red}{F_{n,p,t}^{Piped}}
-        + \sum_{(p, \tilde{p}) \in PCA}\textcolor{red}{F_{p,\tilde{p},t}^{Piped}}
-
-        + \sum_{(s, p) \in SCA}\textcolor{red}{F_{s,p,t}^{Piped}}
-        + \sum_{(r, p) \in RCA}\textcolor{red}{F_{r,p,t}^{Piped}}
-        + \sum_{(p, \tilde{p}) \in PCT}\textcolor{red}{F_{p,\tilde{p},t}^{Trucked}}
+        = \sum_{t \in T}(\sum_{l \in (L-F) | (l, p) \in LLA}\textcolor{red}{F_{l,p,t}^{Piped}}
+        + \sum_{l \in (L-F) | (l, p) \in LLT}\textcolor{red}{F_{l,p,t}^{Trucked}})
 
-        + \sum_{(p, \tilde{p}) \in CCA}\textcolor{red}{F_{p,\tilde{p},t}^{Piped}}
-        + \sum_{(p, \tilde{p}) \in CCT}\textcolor{red}{F_{p,\tilde{p},t}^{Trucked}}
-        + \sum_{(s, p) \in SCT}\textcolor{red}{F_{s,p,t}^{Trucked}})
 
-
-**Piping Cost:** :math:`\forall \textcolor{blue}{(l,\tilde{l}) \in \{PPA, \ldots, CCA\}}, \textcolor{blue}{t \in T}`
+**Piping Cost:** :math:`\forall \textcolor{blue}{l \in (L - O - K)}, \forall \textcolor{blue}{\tilde{l} \in (L - F)}, \forall \textcolor{blue}{(l,\tilde{l}) \in LLA}, \textcolor{blue}{t \in T}`
 
 Piping cost is the total volume of piped water multiplied by the cost for piping.
 
 .. math::
 
     \textcolor{red}{C_{l,\tilde{l},t}^{Piped}}
-        = (\textcolor{red}{F_{l,\tilde{l},t}^{Piped}}
-        + \textcolor{red}{F_{l,\tilde{l},t}^{Sourced})} \cdot \textcolor{green}{\pi_{l,\tilde{l}}^{Pipeline}}
+        = (\textcolor{red}{F_{l \notin F,\tilde{l},t}^{Piped}}
+        + \textcolor{red}{F_{l \in F,\tilde{l},t}^{Sourced})} \cdot \textcolor{green}{\pi_{l,\tilde{l}}^{Pipeline}}
```

Comment:
```suggestion
        + \textcolor{red}{F_{l \in F,\tilde{l},t}^{Sourced}}) \cdot \textcolor{green}{\pi_{l,\tilde{l}}^{Pipeline}}
```
Change color of right paren from red to black.

Reply:
done